freecad_api/main.py

```
@@ -397,622 +158,136 @@ async def convert_to_2d(file: Optional[UploadFile] = File(None), grid_data: Opti
-output_file = os.path.join(output_dir, "drawing.pdf")
-page.exportPDF(output_file)
-
-print(f"PDFを保存しました: {{output_file}}")
 """
             )
 
-        output_file = os.path.join(os.path.dirname(temp_script.name), "drawing.pdf")
-        os.environ["OUTPUT_DIR"] = os.path.dirname(output_file)
-        
         # FreeCADスクリプトを実行
+        output_file = tempfile.NamedTemporaryFile(suffix=".fcstd", delete=False).name
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)